lib/ui/views/groups/group_details_view.dart

```diff
@@ -158,26 +163,32 @@ class _GroupDetailsViewState extends State<GroupDetailsView> {
                 key: _formKey,
                 child: TextFormField(
                   maxLines: 5,
-                  autofocus: true,
+                  onChanged: (emailValue) {
+                    addButtonGlobalKey.currentState
+                        .setAddFunction(emailValue.isNotEmpty);
+                  },
                   decoration: AppTheme.textFieldDecoration.copyWith(
                     hintText: 'Email Ids',
                   ),
                   validator: (emails) => Validators.areEmailsValid(emails)
                       ? null
-                      : 'Enter emails in valid format.',
+                      : 'Enter emails in valid format',
                   onSaved: (emails) =>
                       _emails = emails.replaceAll(' ', '').trim(),
                 ),
               ),
               actions: <Widget>[
                 FlatButton(
                   child: Text('CANCEL'),
-                  onPressed: () => Navigator.pop(context),
-                ),
-                FlatButton(
-                  child: Text('ADD'),
-                  onPressed: () => onAddGroupMemberPressed(context),
+                  onPressed: () {
+                    _emails = null;
+                    Navigator.pop(context);
+                  },
                 ),
+                AddButton(
+                    addFunction: onAddGroupMemberPressed,
+                    context: context,
+                    key: addButtonGlobalKey),
```

Comment:
Can you put comma before brackets and format it again ? It will result in better formatting